### PR TITLE
Feature/GitHub actions

### LINKED ISSUE #84: Add GitHub Actions
Overview for R packages in this [video](https://github.com/FredHutch/VISCfunctions/pull/72/files). Can use `usethis::use_github_actions()`. 

GitHub actions for VISCfunctions: https://github.com/FredHutch/VISCfunctions/pull/72/files

### PR DESCRIPTION
**Pull request by monicagerber**
Added GitHub actions with `usethis::use_github_action("check-standard")`, following recommendation from this [blog](https://www.tidyverse.org/blog/2020/04/usethis-1-6-0/). Closes #84

### CODE REVIEW

**Comment by wfulp** [comment]
@monicagerber should we allow actions on GitHub now?
https://github.com/FredHutch/VISCtemplates/settings/actions

File: .github/workflows/check-standard.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Workflow derived from https://github.com/r-lib/actions/tree/master/examples
+# Need help debugging build failures? Start at https://github.com/r-lib/actions#where-to-find-help
+on:
+  push:
+    branches: [main, master]
```
Comment (wfulp) [inline .github/workflows/check-standard.yaml:5]:
```suggestion
    branches: [main, master, develop]
```
It should be fine adding the checks for develop branch too.

File: .github/workflows/check-standard.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  pull_request:
+    branches: [main, master]
```
Comment (wfulp) [inline .github/workflows/check-standard.yaml:7]:
```suggestion
    branches: [main, master, develop]
```

**Review by wfulp — CHANGES_REQUESTED** [review]
Once we add develop to the checks and allows Github actions we can see if the checks pass.

**Comment by monicagerber** [comment]
I'm not able to change this: 
image

What did you do for VISCfunctions?

**Comment by wfulp** [comment]
I think I got access when I tried to incorporate Travis-CI. It pinged the owner of FredHutch org automatically. I think an email to scicomp@fredhutch.org should work.

**Comment by monicagerber** [comment]
OK, this is allowed now and all actions are enabled. @wfulp let me know if you "approve" the changes.

**Review by wfulp — APPROVED** [review]
All checks passed on a test PR I just tried. 🥳

**Comment by monicagerber** [comment]
Awesome, thanks for your help with this! 🎉

**Pull request merged**